### PR TITLE
Main to develop after 0.3.0

### PR DESCRIPTION
**Pull request by sandeep-ps**
## Description
This PR is to merge changes from the main branch after the 0.3.0 release.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation and CHANGELOG accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by ywkim312** [comment]
there are merge conflicts.

File: src/pages/EQIPPage.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Drawer from "../components/ProgramDrawer";
 import SemiDonutChart from "../components/SemiDonutChart";
 import DataTable from "../components/eqip/EQIPTotalTable";
 import EqipTotalMap from "../components/eqip/EqipTotalMap";
-import chartData from "../data/eqip/EQIP_STATUTE_PERFORMANCE_DATA.json";
+import chartData from "../data/EQIP/EQIP_STATUTE_PERFORMANCE_DATA.json";
```
Comment (sandeep-ps) [inline src/pages/EQIPPage.tsx:9]:
```suggestion
import chartData from "../data/eqip/EQIP_STATUTE_PERFORMANCE_DATA.json";
```
I think this needs to be updated back to small caps to fix the test error.

**Comment by navarroc** [comment]
I see two SNAP legends when you click into the program

**Review by ziyue5 — APPROVED** [review]
Works good to me

**Review by pengyin-shan — APPROVED** [review]
Looks good to me! There's no need to adjust any indent

**Comment by pengyin-shan** [comment]
> I see two SNAP legends when you click into the program



> I see two SNAP legends when you click into the program

Oh sorry, I was thinking to push the fix to this in a separate PR after this merge...SNAP page uses the landing page map component directly. The landing page map used to have the label as a separate png file, which is followed by the SNAP page, but now the landing page map includes a new label as part of it. So now just need to remove the png file in SNAP page as well.

 I now just add my updates to this branch.

**Pull request merged**